test/public/CREATE/CreateX.computeCreateAddress_1Arg.t.sol

```diff
@@ -16,7 +16,7 @@ contract CreateX_ComputeCreateAddress_1Arg_Public_Test is BaseTest {
     /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
 
     modifier whenTheNonceValueDoesNotExceed18446744073709551614(uint64 nonce) {
-        vm.assume(nonce != 0 && nonce < type(uint64).max);
+        vm.assume(nonce != 0 && nonce < type(uint64).max && nonce >= vm.getNonce(createXAddr));
         vm.setNonce(createXAddr, nonce);
```

Comment:
An alternative to consider is leaving the `assume` statements and switching to the `vm.setNonceUnsafe` cheat which does allow decreasing the nonce, which should be safe since these are tests for view methods. That lets us cover a wider range of nonces